core/components/molecules/editableDropdown/__stories__/customRenderer.story.jsx

```diff
@@ -88,7 +96,7 @@ const customCode = `() => {
 
   const customTrigger = (label) => {
     return (
-      <div className="d-flex align-items-center justify-content-between pl-5 pr-4 bg-secondary cursor-pointer w-100">
+      <div className="d-flex customTrigger align-items-center justify-content-between pl-5 pr-4 bg-secondary cursor-pointer w-100">
```

Comment:
Please follow BEM convention for className

Reply:
Done.